packages/cli/src/commands/init/init.ts

```diff
@@ -28,8 +28,10 @@ import {getBunVersionIfAvailable} from '../../tools/bun';
 import {getNpmVersionIfAvailable} from '../../tools/npm';
 import {getYarnVersionIfAvailable} from '../../tools/yarn';
 import {createHash} from 'crypto';
+import execa from 'execa';
 
 const DEFAULT_VERSION = 'latest';
+const packageJson = require('./../../../package.json');
```

Comment:
Is it safe?

Reply:
Maybe sth like this?
https://github.com/react-native-community/cli/blob/dea06427cedcfddd5170509e8dad797f38c356c2/packages/cli-config/src/findDependencies.ts#L11-L13

Reply:
Since we added reading RN version from `node_modules`, we don't need to read `package.json`.